### ISSUE TITLE
Prettier / eslint: vue files

### ISSUE THREAD

**Issue by pascalgrimaud** [jhipster/jhipster-vuejs#608]
I think it would be cool to have prettier / eslint for all vue files
Probably related to https://github.com/jhipster/jhipster-vuejs/issues/597 and https://github.com/jhipster/jhipster-vuejs/pull/601

**Comment by atomfrede** [comment]
Yes, I will use the configuration created by vue-cli for eslint + prettier, so I guess it will be solved with that one.

**Comment by hermitmaster** [comment]
@atomfrede Thanks for taking this and #597 on!
